microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -218,6 +222,10 @@ If you're debugging Office Add-ins, open the add-in source code in a separate in
 
    ![Run and Debug](./debug-visual-studio-code-images/attach-uwp.png)
 
+<!-- ====================================================================== -->
+## Source maps with `WebResourceRequested` event or virtual host name mapping
+
+Source maps are needed to debug the source code of compiled content like transpiled JavaScript (e.g. TypeScript, minified JavaScript) or CSS (e.g. SASS, SCSS). WebView2 does not load source maps referenced by content which was loaded using either [WebResourceRequested event](../concepts/working-with-local-content.md#loading-local-content-by-handling-the-webresourcerequested-event) or [virtual host name mapping](../concepts/working-with-local-content.md#loading-local-content-by-using-virtual-host-name-mapping). See details and solutions for [WebResourceRequested event here](../concepts/working-with-local-content.md#source-maps-with-virtual-host-name-mapping), and for [virtual host name mapping here](../concepts/working-with-local-content.md#source-maps-with-virtual-host-name-mapping).
 
```

Comment:
```suggestion
```
reverting this h2 section addition, per [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3384/files#r1965590535) 
@mikosg @dhveerap 
 in `debug-visual-studio-code.md`

Reply:
might have to entirely delete this comment, might be seeing Outdated markdown, i think we're supposed to keep this h2